Roys/app/src/main/java/com/dwg_karrier/roys/ScriptedURL.java

```diff
@@ -42,7 +47,16 @@ public boolean getIsRead() {
     return isRead;
   }
 
+  public String getTitle() {
+    return crawler.getTitle();
+  }
+
+  public String getContent() {
+    return crawler.getContent();
+  }
+
   public int getWordCount() {
+    wordCount = crawler.getWordCount();
```

Comment:
It was already declared when I pull Soyee's code. I guess it is  because she need to make test data which was added using `ScriptedURL(String url, int isRead, int wordCount)`.

When I revise the test code, I recognized that wordCount is not needed.
However after I read Sunju's DB schema, I found `public void setWordCount(String url, int wordCount)` which might be used later. 
Although we talked about our structure to not keep wordCount in DB, I left this just in case. Time consumes quite a lot when switching from MainActivity to ListActivity, using wordCount info right away might reduce the time. 

(Calling Crawler makes time consuming so we need to talk about **How to use Crawler efficiently**)